src/Sylius/Behat/Context/Setup/ChannelContext.php

```diff
@@ -263,6 +277,21 @@ public function itsRequiredAddressInTheCheckoutIs(string $type): void
         $this->channelManager->flush();
     }
 
+    /**
+     * @Given the store also operates in :localeCode locale
+     */
+    public function theStoreAlsoOperatesInLocale(string $localeCode): void
+    {
+        $locale = $this->localeRepository->findOneBy(['code' => $localeCode]);
+        Assert::notNull($locale);
```

Comment:
Use here a locale transformer:
```suggestion
    /**
     * @Given the store also operates in :locale locale
     */
    public function theStoreAlsoOperatesInLocale(LocaleInterface $locale): void
    {
```